1-exercises/B-highlight-words/script.js

```diff
@@ -1,9 +1,38 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  
+ const divEl = document.querySelector("#content")
+ const selectEl = document.createElement("select")
+ const paraEl = document.createElement("p")
+
+ const splitPara = paragraph.split(" ")
+ for(let word of splitPara){
+    const spanEl  = document.createElement("span")
+      spanEl.innerText = word;
+      spanEl.style.padding = 3 + "px";
```

Comment:
```suggestion
      spanEl.style.padding = "3px";
```
Just use the string literal `"3px"` for the padding here ( or create a CSS class with this padding value )